lte/gateway/c/oai/tasks/mme_app/mme_app_itti_messaging.c

```diff
@@ -148,8 +148,20 @@ int mme_app_send_s11_release_access_bearers_req(
   release_access_bearers_request_p->originating_node = NODE_TYPE_MME;
 
   message_p->ittiMsgHeader.imsi = ue_mm_context->emm_context._imsi64;
+  if (pdn_connection->route_s11_messages_to_s8_task) {
+    OAILOG_INFO_UE(
+        LOG_MME_APP, ue_mm_context->emm_context._imsi64,
+        "Send Release Access Bearer Req for teid to sgw_s8 task " TEID_FMT "\n",
+        ue_mm_context->mme_teid_s11);
+    rc = send_msg_to_task(&mme_app_task_zmq_ctx, TASK_SGW_S8, message_p);
```

Comment:
return value rc is never checked I think? Here and elsewhere. Can send_msg_to_task fail? How should that be handled if so.

Reply:
send_msg_to_task always returns "0" which is equivalent to "RETURNok" so I will remove the rc.